### PR TITLE
UPDATE: New import function DEG process.

### PR DESCRIPTION
**Pull request by esguerra**
- New import function to process the excel files with
data from Inoue and Bouvier as processed by DEG.
- New fields in ProteinGProteinPair table
- Migration for new fields in the ProteinGProteinPair table
- Commented out for the moment values for ProteinArrestinPair table in models.
will be needed soon.
- Small comment out of print statements in sequence_signature.py
- Start refactoring url dispatcher to Django 2 before deprecation.

url()¶
url(regex, view, kwargs=None, name=None)¶
This function is an alias to django.urls.re_path(). It’s likely to be deprecated in a future release.

### CODE REVIEW

File: seqsign/sequence_signature.py [hunk 1]
```diff
@@ -1283,7 +1283,7 @@ def score_protein_class(self, pclass_slug='001', signprot=False):
             protein_scores[pcf] = (score, nscore)
             protein_signature_match[pcf] = signature_match
             p_end = time.time()
-            print("Time elapsed for {}: ".format(pcf.protein.entry_name), p_end - p_start)
+            # print("Time elapsed for {}: ".format(pcf.protein.entry_name), p_end - p_start)
```
Comment (AJK-dev) [inline seqsign/sequence_signature.py:1286]: Codacy highlighted that by commenting this you can also comment the p_start and p_end (sa

File: signprot/urls.py [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-    url(r'^ajax/interface/(?P<slug>[-\w]+)/$', views.ajaxInterface, name='ajaxInterface'),
-    url(r'^structure/(?P<pdbname>[-\w]+)/$', views.StructureInfo, name='StructureInfo'),
-    url(r'^family/(?P<slug>[-\w]+)/$', views.familyDetail, name='familyDetail'),
-    url(r'^matrix[/]?$', views.InteractionMatrix, name='InteractionMatrix'),
-    url(r'^matrix/seqsig/', views.IMSequenceSignature, name='SequenceSignature'),
-    url(r'^matrix/sigmat/', views.IMSignatureMatch, name='SignatureMatch'),
-    url(r'^matrix/render_sigmat/$', views.render_IMSigMat, name='renderSignatureMatch'),
-    url(r'^pdbtabledata', PdbTableData, name='pdbtreedata'),
-    url(r'^(?P<slug>[-\w]+)/$', views.signprotdetail, name='signprotdetail'),
+    path('', views.BrowseSelection.as_view(), name='index'),
```
Comment (AJK-dev) [inline signprot/urls.py:8]: Good point. I guess we should also convert some "re_path" to "path" calls.

**Review by AJK-dev — APPROVED** [review]
Looks good. As discussed the redundancy will be removed later upon finishing the whole thing.

Was just starting to integrate Codacy for the protwis repo for code quality checks:
https://app.codacy.com/gh/protwis/protwis/pullRequest?prid=6521496

Codacy is happy with you 😁

File: signprot/urls.py (hunk 2 above)
Comment (esguerra) [reply signprot/urls.py:8]: True. Better to convert re_path to path.
Just don't know how the "regexes" work with path alone yet. Will look into it.

**Pull request merged**